README.md

```diff
@@ -120,14 +184,100 @@ This folder contains examples that illustrate the use of Amazon Braket Hybrid Jo
 
-* [**Bring your own containers to Braket Jobs**](examples/hybrid_jobs/3_Bring_your_own_container/bring_your_own_container.ipynb)
+* [**Bring your own containers to Braket Hybrid Jobs**](examples/hybrid_jobs/3_Bring_your_own_container/bring_your_own_container.ipynb)
 
     This notebook demonstrates the use of the Bring-Your-Own-Container (BYOC) functionality of Braket Hybrid Jobs. While Amazon Braket has pre-configured environments which support most use cases of Braket Hybrid Jobs, BYOC enables you to define fully customizable environments using Docker containers. You will learn how to use BYOC, including preparing a Dockerfile, creating a private Amazon Elastic Container Registry (ECR), building the container, and submitting a Braket Hybrid Job using the custom container.
-    
+
+* [**Embedded simulators in Braket Hybrid Jobs**](examples/hybrid_jobs/4_Embedded_simulators_in_Braket_Hybrid_Jobs/Embedded_simulators_in_Braket_Hybrid_Jobs.ipynb)
+
+    This notebook shows how to use embedded simulators in Braket Hybrid Jobs. An embedded simulator is a local simulator that runs completely within a hybrid job instance, i.e., the compute resource that is running your algorithm script. In contrast, on-demand simulators, such as SV1, DM1, or TN1, calculate the results of a quantum circuit on dedicated compute infrastructure on-demand by Amazon Braket. Hybrid workloads usually consist of iterations of quantum circuit executions and variational parameter optimizations. By using embedded simulators, we keep all computations in the same environment. This allows the optimization algorithm to access advanced features supported by the embedded simulator.
+
+* [**Parallelize training for Quantum Machine Learning**](examples/hybrid_jobs/5_Parallelize_training_for_QML/Parallelize_training_for_QML.ipynb)
+
+    This notebook introduces using data parallelism for Quantum Machine Learning (QML) workloads.
+
+* [**QN-SPSA optimizer using an Embedded Simulator**](examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb)
+
+    This notebook demonstrates how to implement and benchmark the QN-SPSA optimizer, a novel quantum optimization algorithm.
+
+* [**Running Jupyter notebooks as a Hybrid Job**](examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/Running_notebooks_as_hybrid_jobs.ipynb)
+
+    This tutorial is a step-by-step guide for running a Jupyter notebook as a Hybrid Job.
+
+* [**Creating Hybrid Job Scripts**](examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb)
+
+    This notebook shows an alternate way to create a Hybrid Job - without using a @hybrid_job decorator - that may be useful in some circumstances, such as using older versions of Python. 
```

Comment:
Again, I'm leaving it a little vague here so we don't forget to update it and give wrong information. Customers can always look up what versions we support, but giving them the wrong information is worse than having to look it up.

Reply:
This sentence makes use of too many clauses in the description. Breaking it up past "decorator" would be better.

Reply:
I wanted to convey that the main difference is that the decorator is not used in this notebook because we want to make it available for customers who are using a python version that's not the same one we have in the NBIs/containers (3.10, for now).

Reply:
I just think it reads slightly smoother as two sentences such as:
```
This notebook shows an alternate way to create a Hybrid Job without using a @hybrid_job decorator. The demonstrated method may be useful in some circumstances, such as using older versions of Python. 
```
I don't think this is a blocking issue to get this merged.